app/Http/Controllers/Teams/MembersController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Teams;
+
+use App\Http\Controllers\Controller;
+use App\Models\Team;
+use App\Models\TeamMember;
+use Symfony\Component\HttpFoundation\Response;
+
+class MembersController extends Controller
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->middleware('auth');
+    }
+
+    public function destroy(string $teamId, string $userId): Response
```

Comment:
from what I gather the type will always be string (or nullable string if optional)